src/commands/read.ts

```diff
@@ -64,9 +71,11 @@ export const handler = async (argv: Arguments<Options>): Promise<void> => {
 
     if (format === "pretty") {
       console.table(res.results);
+    } else if (format === "table") {
+      console.log(JSON.stringify(transformTableData(res.results)));
     } else {
       const out = res;
-      console.dir(out, { depth: null });
+      console.log(JSON.stringify(out, null, "\t"));
```

Comment:
I still think we only want to output the res.results here, right? Otherwise, we should _probably_ include (as a separate feature and flag perhaps) the ability to include `--meta` information?

Reply:
`--verbose`? this is a tricky one... i think the easiest thing would be to let consumers extract the actual results with `jq`

Reply:
Ok, I'm cool with that for sure. Agreed. This is less clear for the "pretty" and "table" options, but for objects 👍